packages/appium/lib/main.js

```diff
@@ -170,6 +172,18 @@ function areServerCommandArgs(args) {
 async function init(args) {
   const appiumHome = args?.appiumHome ?? (await resolveAppiumHome());
 
+  // this is here so extensions installed in e.g., `~/.appium` can find the peer dependency
+  // of `appium`, which lives wherever it lives (see `rootDir`).
+  if (!(await hasAppiumDependency(appiumHome))) {
+    try {
+      await fs.symlink(rootDir, path.join(appiumHome, 'node_modules', 'appium'));
+    } catch (err) {
```

Comment:
is rootDir pointing to the root of the global Appium installation?
Maybe then it would make sense to put a symlink to `$HOME/.node_modules` according to https://nodejs.org/api/modules.html#loading-from-the-global-folders